src/index.js

```diff
@@ -1 +1,91 @@
+const weatherPart = wrapper.querySelector('.weather-part');
+const wIcon = weatherPart.querySelector('img');
+const arrowBack = wrapper.querySelector('header i');
+const apiKey = '5b1f0a980bd35cdef966b7da15d9b7f5';
+
+let api;
+
+inputField.addEventListener('keyup', (e) => {
+  if (e.key === 'Enter' && inputField.value !== '') {
+    requestApi(inputField.value);
+  }
+});
+
+locationBtn.addEventListener('click', () => {
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(onSuccess, onError);
+  } else {
+    alert('Your browser not support geolocation api');
+  }
+});
+
+function requestApi(city) {
+  api = `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${apiKey}`;
+  fetchData();
+}
+
+function onSuccess(position) {
+  const { latitude, longitude } = position.coords;
+  api = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&units=metric&appid=${apiKey}`;
+  fetchData();
+}
+
+function onError(error) {
+  infoTxt.innerText = error.message;
+  infoTxt.classList.add('error');
+}
+
+function fetchData() {
+  infoTxt.innerText = 'Getting weather details...';
+  infoTxt.classList.add('pending');
+  fetch(api).then((res) => res.json()).then((result) => weatherDetails(result)).catch(() => {
+    infoTxt.innerText = 'Something went wrong';
+    infoTxt.classList.replace('pending', 'error');
+  });
+}
+
+function weatherDetails(info) {
+  if (info.cod === '404') {
+    infoTxt.classList.replace('pending', 'error');
+    infoTxt.innerText = `${inputField.value} isn't a valid city name`;
+  } else {
+    const city = info.name;
+    const { country } = info.sys;
+    const { description, id } = info.weather[0];
+    // eslint-disable-next-line camelcase
+    const { temp, feels_like, humidity } = info.main;
+
+    if (id === 800) {
+      wIcon.src = 'icons/clear-day.svg';
+    } else if (id >= 200 && id <= 232) {
+      wIcon.src = 'icons/thunderstorms.svg';
+    } else if (id >= 600 && id <= 622) {
+      wIcon.src = 'icons/snow.svg';
+    } else if (id >= 701 && id <= 781) {
+      wIcon.src = 'icons/haze.svg';
+    } else if (id >= 801 && id <= 804) {
+      wIcon.src = 'icons/cloudy.svg';
+    } else if ((id >= 500 && id <= 531) || (id >= 300 && id <= 321)) {
+      wIcon.src = 'icons/rain.svg';
+    }
+
+    weatherPart.querySelector('.temp .numb').innerText = Math.floor(temp);
+    weatherPart.querySelector('.weather').innerText = description;
+    weatherPart.querySelector('.location span').innerText = `${city}, ${country}`;
+    weatherPart.querySelector('.temp .numb-2').innerText = Math.floor(feels_like);
+    weatherPart.querySelector('.humidity span').innerText = `${humidity}%`;
+    infoTxt.classList.remove('pending', 'error');
+    infoTxt.innerText = '';
+    inputField.value = '';
+    wrapper.classList.add('active');
+  }
+}
+
+arrowBack.addEventListener('click', () => {
+  wrapper.classList.remove('active');
+});
```

Comment:
- [ ] Your app should have a toggle control to determine if the result will be shown in Celsius or Fahrenheit (There are two valid solutions: you can make two different requests for the Celsius data and the Fahrenheit data OR make only one request and use math to convert it to the other unit) 
- [ ] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example: 

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

- [ ] Please, make use of arrow functions instead of the `function` keyword. 
- [ ] Please include in your README details of the API(s) your app consumes.